src/Components/alert/Alert.css

```diff
@@ -0,0 +1,71 @@
+:root {
+  --color-1: #f44336;

```

Comment:
I have a suggestion, instead of color-1,2 variables, you can add more descriptive names like red-color, or primary-color